examples/storm-elasticsearch-examples/src/main/java/org/apache/storm/elasticsearch/bolt/EsIndexTopology.java

```diff
@@ -34,52 +39,128 @@
 import org.apache.storm.tuple.Fields;
 import org.apache.storm.tuple.Values;
 
-public class EsIndexTopology {
+/**
+ * Demonstrates an ElasticSearch Strom topology.
+ * @author unknown
+ */
+public final class EsIndexTopology {
 
+    /**
+     * The id of the used spout.
+     */
     static final String SPOUT_ID = "spout";
```

Comment:
I think these should be private, that way checkstyle hopefully won't complain about missing comments.